src/gui/sdl_mapper.cpp

```diff
@@ -48,6 +48,15 @@
 #include "timer.h"
 #include "video.h"
 
+//  Status Colors
+//  ~~~~~~~~~~~~~
+//  NFPA 79 standard for illuminated status indicators:
+//  (https://www.nfpa.org/assets/files/AboutTheCodes/79/79-A2002-rop.pdf pp.1588-1593)
+//
+constexpr uint32_t marginal_color = 0x00'ff'67'00; // Amber for marginal conditions
+constexpr uint32_t on_color = 0x00'00'01'00;       // Green for on/ready/in-use
+constexpr uint32_t off_color = 0x00'00'00'00;      // Black for off/stopped/not-in-use
+
```

Comment:
Why not use a struct with rgb fields, and use accordingly? Would avoid the bitwise operations later.

Reply:
If there's a strong desire for users to customize these colors (I hope not.. but if they do), I would prefer to go with a single `set_Hex()` value in the conf setting, which is a lot easier to parse than a comma-seprarated string.  We could directly assign the `get_Hex()` value to a uint32_t, and then bounds-check it.
Hex has become the dominant color format - even Google's picker uses it by default:

![2022-01-20_11-29](https://user-images.githubusercontent.com/1557255/150409378-70af0e3b-44e5-4c10-abe6-d334475f8a57.png)

Reply:
But that said - for the scope of the PR, it probably makes more sense to use the RGB struct - and then only move to hex & unpacking if there's a strong push for more features in this area.

Reply:
There's also no reason why you couldn't give the rgb24 struct a constructor that parses hex color values too.